src/components/search/_search.scss

```diff
@@ -5,6 +5,7 @@ $include-html: false !default;
 
   .mint-search {
     @include component;
+    overflow: visible;
```

Comment:
`component` sets `overflow: hidden` we don't want that for search because it cuts the shadow of the `mint-input--subtle-border`